app/uk/gov/hmrc/merchandiseinbaggage/model/api/Declaration.scala

```diff
@@ -195,6 +198,45 @@ case class Declaration(declarationId: DeclarationId,
       eori = eori.obfuscated,
       journeyDetails = journeyDetails.obfuscated
     )
+
+  def toEmailInfo(bfEmail: String): DeclarationEmailInfo = {
+    val templateId = if (declarationType == DeclarationType.Import) "mods_import_declaration" else "mods_export_declaration"
+    val goodsParams = declarationGoods.goods.zipWithIndex.map { goodsWithIdx =>
+      val (goods, idx) = goodsWithIdx
+      val countryOrDestKey =  if (declarationType == DeclarationType.Import) s"goodsCountry_$idx" else s"goodsDestination_$idx"
+      Map(
+        s"goodsCategory_$idx" -> goods.categoryQuantityOfGoods.category,
+        s"goodsQuantity_$idx" -> goods.categoryQuantityOfGoods.quantity,
+        countryOrDestKey -> goods.countryOfPurchase,
+        s"goodsPrice_$idx" -> goods.purchaseDetails.toString,
+      )
+    }.reduce(_ ++ _)
+
+    val commonParams = Map(
+      "nameOfPersonCarryingGoods" -> nameOfPersonCarryingTheGoods.toString,
+      "declarationReference" -> mibReference.value,
+      "dateOfDeclaration" -> dateOfDeclaration.toString,
+      "eori" -> eori.value
+    )
+
+    val calculationParams = Map(
+      "customsDuty" -> "???",
+      "vat" -> "???",
+      "total" -> "???"
+    )
+
+    val allParams =
+      if (declarationType == DeclarationType.Import)
+        goodsParams ++ commonParams ++ calculationParams
+      else
+        goodsParams ++ commonParams
+
+    DeclarationEmailInfo(
+      Seq(email.email, bfEmail),
```

Comment:
I think we only want to send to borderforce for import, they aren't concerned with export as you don't go through borderforce red/green channels when leaving the country